guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated mints**, **Federated single guardian mints**, and **Solo mints (Unfederated mints)**, each with its strengths and trade-offs.
+
+## Role of guardians
+Guardians are participants who collaboratively manage a federations operations. They secure the federation's funds using multi-sig wallets, validate payments through consensus, and control ecash issuance and redemption.
+
+## Role of gateways
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   retina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   alt-text = "Network diagram showing how gateways connect federations (represented by blue dots) with the lightning network (represented by orange dots). The nodes are linked by dashed lines, visualizing the interconnected structure of federations and lightning nodes."
+   caption = "Gateways connect a federation’s ecash and bitcoin via the lightning network."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+A gateway is a service that facilitates interactions between the federation (which operates largely off-chain) and the lightning network. A gateway acts as a bridge, enabling users within the federation to make payments to and receive payments on the lightning network.
```

Comment:
I would get rid of "which operates largely off-chain". Maybe instead you could add a paragraph about why gateways are necessary, describing that without them, users could only operate entirely off-chain when transacting with other users within the same federation (through direct ecash swaps).